clients/lsp-terraform.el

```diff
@@ -93,6 +93,29 @@
                   :priority 1
                   :server-id 'tfmls))
 
+(defun lsp-terraform-ls-validate ()
+  "Execute terraform validate on `default-directory'."
```

Comment:
the comment is not accurate - you are executing it over project root. `default-directory` is the current directory.

Reply:
Ah, it was a comment from the old implementation. Fixed it, thanks!